tests/unit/resources/networking/test_logical_interconnect_groups.py

```diff
@@ -40,15 +40,16 @@ def setUp(self):
     def test_get_all_called_once(self, mock_get_all):
         filter = 'name=TestName'
         sort = 'name:ascending'
+        scopeUris = 'TestScope'
```

Comment:
Better to give scopeUris as uri because its misleading. e.g. '/rest/scopes/cd237b60-09e2-45c4-829e-082e318a6d2a'